assets/css/pages/about-us/header.css

```
@@ -0,0 +1,472 @@
+    .container {
+        width: 0;
+    }
+
+    .navigation-menu {
+        display: none;
+    }
+
+    #carousel {
+        padding: 25px 0;
+        top: 15px;
+        text-align: center;
+        position: relative;
+        left: 0;
+    }
+
+    .hamburger-button {
+        display: inline-block;
+    }
+
+    .computer {
+        height: 60vh;
+        position: absolute;
+        z-index: 0;
+        top: 80px;
+        left: 9%;
+    }
+
+    .computer-container {
+        width: 100vw;
+        height: 100vh;
+        background-size: 280px;
+        background-position-y: 20px;
+    }
+
+    .image-holder {
+        display: block;
+        margin-bottom: 0;
+        width: 175px;
+        height: 175px;
+    }
+
+    .carousel-item-img {
+        display: block;
+        width: 175px;
+        margin-right: auto;
+        margin-left: auto;
+    }
+
+    .description-bg .name {
+        font-size: 36px;
+    }
+
+    .description-bg .desc-title {
+        font-size: 24px;
+    }
+
+    .description-bg .skills {
+        font-size: 20px;
+    }
+
+    .description-bg .desc-links {
+        padding-right: 6px;
+    }
+
+    .description-bg .description {
+        font-size: 16px;
+        padding: 6px;
+        position: relative;
+        top: 0;
+        line-height: 1.3;
+    }
+
+    .description-bg .desc-links-before {
+        display: none;
+    }
+
+    .description-bg .desc-links-after {
+        display: block;
+        position: absolute;
+        bottom: 20px;
+        right: 15px;
+    }
+
+    .desc-links .mdi {
+        font-size: 30px;
+    }
+
+    .hero,
+    .space-bg,
+    .space-bg-overlay {
+        height: calc(-39.370vw + 1602.559px);
+    }
+
+    .description-bg {
+        padding: calc(2.22vh - 3.333px);
+        height: calc(-39.370vw + 847.559px);
+        top: calc((-39.370vw + 1602.559px) - (-39.370vw + 847.559px));
+    }
+
+    .description-sizer {
+        height: calc((-39.370vw + 847.559px) / 2);
+    }
+}
+
+
+@media only screen and (max-width: 750px) {
+    #carousel {
+        top: -10px;
+    }
+
+    .computer {
+        height: 50vh;
+        left: 15%;
+    }
+
+    .computer-container {
+        width: 100vw;
+        height: 100vh;
+        background-size: 190px;
+        background-position-y: 0;
+    }
+
+    .image-holder {
+        display: block;
+        margin-bottom: 0;
+        width: 120px;
+        height: 120px;
+    }
+
+    .carousel-item-img {
+        display: block;
+        width: 120px;
+        margin-right: auto;
+        margin-left: auto;
+    }
+
+    .hero,
+    .space-bg,
+    .space-bg-overlay {
+        height: calc(-39.370vw + 1402.559px);
+    }
+
+    .description-bg {
+        padding: calc(2.22vh - 3.333px);
+        height: calc(-39.370vw + 907.559px);
+        top: calc((-39.370vw + 1402.559px) - (-39.370vw + 907.559px));
+    }
+
+    .description-sizer {
+        height: calc((-39.370vw + 907.559px) / 2);
+    }
+
+}
```

*NITPICK* Github complains when you don't have a new line at the end of the file
```suggestion
}

```